### PR TITLE
Edits to the openshift_model doc

### PR DESCRIPTION
**Pull request by bfallonf**
@liggitt This PR is in response to #480 . I didn't make any significant changes, but can I ask for a quick review?

Thanks.

### CODE REVIEW

File: architecture/core_objects/openshift_model.adoc [hunk 1]
```diff
@@ -514,18 +520,20 @@ Sample Identity:
 }
 ----
 
-<1> Identity name must be in the form providerName:providerUserName
-<2> The name of the identity provider
```
Comment (liggitt) [inline architecture/core_objects/openshift_model.adoc]: Is it correct to put a period at the end of a fragment?

File: architecture/core_objects/openshift_model.adoc [hunk 2]
```diff
@@ -628,10 +639,12 @@ Sample OAuthClient:
 }
 ----
 
-<1> Used as the `client_id` parameter in OAuth requests
-<2> When `true`, unauthenticated requests to `/oauth/authorize` will be result in `WWW-Authenticate` challenges, if supported by the configured authentication methods.
-<3> Used as the `client_secret` parameter in an authorization code flow
-<4> One or more absolute URIs. The `redirect_uri` parameter sent with authorization requests must be prefixed by one of the specified `redirectURIs`
+<1> Used as the `client_id` parameter in OAuth requests.
+<2> When `true`, unauthenticated requests to `/oauth/authorize` will result in
+`WWW-Authenticate` challenges, if supported by the configured authentication
```
Comment (liggitt) [inline architecture/core_objects/openshift_model.adoc]: do newlines get rendered correctly with annotations?

File: architecture/core_objects/openshift_model.adoc [hunk 3]
```diff
@@ -560,11 +568,14 @@ Sample User:
...
-Creating, updating, or deleting a UserIdentityMapping modifies the corresponding fields on the Identity and User objects.
+Creating, updating, or deleting a UserIdentityMapping modifies the corresponding
+fields in the Identity and User objects.
 
-An Identity can only map to a single User (so that logging in as a particular identity unambiguously determines the User).
+An Identity can only map to a single User, so logging in as a particular
+identity unambiguously determines the User.
 
-A User can have multiple Identities mapped to it (so multiple login methods can identify the same User).
+A User can have multiple Identities mapped to it. Multiple login methods can
```
Comment (liggitt) [inline architecture/core_objects/openshift_model.adoc]: "This allows multiple login methods to identify the same User."

File: architecture/core_objects/openshift_model.adoc (hunk 2 above)
Comment (bfallonf) [inline architecture/core_objects/openshift_model.adoc]: @liggitt It should render correctly. There's a few examples of it being ok.

File: architecture/core_objects/openshift_model.adoc (hunk 1 above)
Comment (bfallonf) [inline architecture/core_objects/openshift_model.adoc]: Checked into this, and yeah, needed further edits. I changed the instances around and it should be both correctly worded and punctuated.

File: architecture/core_objects/openshift_model.adoc (hunk 3 above)
Comment (bfallonf) [inline architecture/core_objects/openshift_model.adoc]: Done.

**Comment by bfallonf** [comment]
@liggitt Thanks for the review. Comments inline.

**Comment by adellape** [comment]
LGTM

**Pull request merged**